core/src/main/resources/org/apache/spark/ui/static/sorttable.js

```diff
@@ -100,20 +100,20 @@ sorttable = {
             this.removeChild(document.getElementById('sorttable_sortfwdind'));
             sortrevind = document.createElement('span');
             sortrevind.id = "sorttable_sortrevind";
-            sortrevind.innerHTML = stIsIE ? '&nbsp<font face="webdings">5</font>' : '&nbsp;&#x25BE;';
+            sortrevind.innerHTML = stIsIE ? '&nbsp<font face="webdings">5</font>' : '&nbsp;&#x25B4;';
```

Comment:
So this is now an up arrow not down arrow -- just making sure I read the intend of the change correctly